common/containers/Tabs/Contracts/components/Interact/components/InteractExplorer/index.tsx

```diff
@@ -67,25 +68,102 @@ class InteractExplorerClass extends Component<Props, State> {
     contractFunctions: {}
   };
 
+  public firstTimer: any;
+  public secondTimer: any;
+
+  public refScrollTo: any;
+
   public state: State = {
     selectedFunction: null,
     inputs: {},
     outputs: {}
   };
 
+  public setParams = () => {
+    const params = this.getParamsFromUrl();
+    let count = 0;
+    Object.keys(params).map((index: any) => {
+      const item = params[index];
+      if (index.substr(0, 5) === 'param') {
+        count++;
+        const element: any = document.getElementById(index);
```

Comment:
Can you explain what this method is attempting to do and why this `getElementById` is needed?

Reply:
Just grab the params from the url and set them in the function's input fields according to their respective numbers. It needs getElementById in order to get the input and set the value to that input

Reply:
Hi @blurpesec. Any news?